docs/source/demo.rst

```diff
@@ -95,7 +95,7 @@ equivalent command using:
 .. code-block:: bash
 
     $ source ga4gh-env/bin/activate
-    (ga4gh-env) $ ga4gh_client readgroupsets-search --datasetIds XXXX http://localhost:8000/current
+    (ga4gh-env) $ ga4gh_client readgroupsets-search --dataset_ids XXXX http://localhost:8000/current
```

Comment:
Yes, the `--dataset_ids` parameter works when running under the `snake_case` branch:
```
$ python client_dev.py variantsets-search --dataset_ids dataset1 http://localhost:8000/current
1kg-phase1
1kg-phase3
$
```